_includes/header.html

```diff
@@ -1,17 +1,17 @@
-<header class="pldoc-header-site">
+<header class="pldoc-header-site" role="banner" aria-labelledby="pldoc_header_site_title">
     {% if page.title == "Home" %}
-    <h1 class="pldoc-header-site-title">
+    <h1 class="pldoc-header-site-title" id="pldoc_header_site_title">
         <a class="pldoc-link title-name" href="{{ site.baseurl }}">
             {{ site.title }}
         </a>
-        <span class="title-description copy copy-base">{{ site.tagline }}</span>
     </h1>
+    <span class="title-description copy copy-base">{{ site.tagline }}</span>
     {% else %}
     <h2 class="pldoc-header-site-title">
         <a class="pldoc-link title-name" href="{{ site.baseurl }}">
             {{ site.title }}
         </a>
-        <span class="title-description copy copy-base">{{ site.tagline }}</span>
     </h2>
+    <span class="title-description copy copy-base">{{ site.tagline }}</span>
     {% endif %}
```

Comment:
Also, your ``<span class="title-description copy copy-base">{{ site.tagline }}</span>`` line can be moved outside of the Jekyll if statement to only include it once (rather than twice in the source code). Please simplify.